server/index.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express')
+const path = require('path')
+const cookieParser = require('cookie-parser')
+const Bunyan = require('bunyan')
+const config = require('../config')
+
+const log = Bunyan.createLogger({ name: 'dev-center-server' })
+const app = express()
+
+const isProduction = process.env.NODE_ENV === 'production'
+
+log.info('Starting server')
+
+app.use(cookieParser())
+
+// Add security headers to all responses
+app.use((req, res, next) => {
+  if (!config.env.disableXXSSProtection) {
+    res.setHeader('X-XSS-Protection', 1)
+  }
+  if (!config.env.disableXContentTypeOptions) {
+    res.setHeader('X-Content-Type-Options', 'nosniff')
+  }
+  if (!config.env.disableXFrameOptions) {
+    res.setHeader('X-Frame-Options', 'deny')
+  }
+  if (!config.env.disableHSTS) {
+    res.setHeader(
+      'Strict-Transport-Security',
+      'max-age=86400; includeSubDomains'
+    )
+  }
+  next()
+})
+
+// Remove trailing slashes
+app.get('*', (req, res, next) => {
+  if (/.+\/$/.test(req.path)) {
+    res.redirect(req.path.replace(/\/$/, ''))
+  } else {
+    next()
+  }
+})
+
+// Dev Center
+
+const isDirectory = path => !/\w+\.\w+$/.test(path)
+app.use(
+  /^\/developers/,
```

Comment:
:bulb: Ah, per the note in the ticket, I'd prefer if this server didn't have to know anything about the `/developers` path. Perhaps there's a technical constraint I'm missing?